frontend/src/app/DeploySmokeTest.test.tsx

```diff
@@ -1,30 +1,56 @@
 import { render, screen, waitFor } from "@testing-library/react";
 import { FetchMock } from "jest-fetch-mock";
 
-import DeploySmokeTest from "./DeploySmokeTest";
+import DeploySmokeTest, {
+  APP_STATUS_FAILURE,
+  APP_STATUS_LOADING,
+  APP_STATUS_SUCCESS,
+  OKTA_STATUS_LOADING,
+  OKTA_STATUS_SUCCESS,
+} from "./DeploySmokeTest";
+import { generateBackendApiHealthResponse } from "./deploySmokeTestTestConstants";
 
 describe("DeploySmokeTest", () => {
   beforeEach(() => {
     (fetch as FetchMock).resetMocks();
   });
 
   it("renders success when returned from the API endpoint", async () => {
-    (fetch as FetchMock).mockResponseOnce(JSON.stringify({ status: "UP" }));
+    (fetch as FetchMock).mockResponseOnce(
+      JSON.stringify(generateBackendApiHealthResponse("UP"))
+    );
+    (fetch as FetchMock).mockResponseOnce(
+      JSON.stringify(generateBackendApiHealthResponse("DOWN"))
+    );
 
     render(<DeploySmokeTest />);
     await waitFor(() =>
-      expect(screen.queryByText("Status loading...")).not.toBeInTheDocument()
+      expect(screen.queryByText(APP_STATUS_LOADING)).not.toBeInTheDocument()
     );
-    expect(screen.getByText("Status returned success :)"));
+    expect(screen.getByText(APP_STATUS_SUCCESS));
   });
 
   it("renders failure when returned from the API endpoint", async () => {
-    (fetch as FetchMock).mockResponseOnce(JSON.stringify({ status: "DOWN" }));
+    (fetch as FetchMock).mockResponse(
+      JSON.stringify(generateBackendApiHealthResponse("DOWN"))
```

Comment:
I could've been more clear here sorry - we're hitting `/actuator/health/backend-and-db-smoke-test` when getting app status so the shape of that response would be different than what's generated by this method

Reply:
ahh so in that case what used to be here is actually the shape of the response 😅 

https://github.com/CDCgov/prime-simplereport/assets/29645040/18c03dae-3c43-4831-a9b7-587007e05457

Reply:
no this is my bad! swapped it back to what it used to be but let me know if i've still misunderstood you 😅

Reply:
nope that's right! thank you